db/migrate/20191020172643_add_name_to_game.rb

```diff
@@ -0,0 +1,4 @@
+class AddNameToGame < ActiveRecord::Migration[5.2]
+  def change
+  end
```

Comment:
Do a `rails db:rollback` and then add the following:
`add_column :games, :name, :string`

After you can do `rails db:migrate` again